src/coprocessor/tracker.rs

```diff
@@ -353,14 +353,14 @@ impl<E: Engine> Tracker<E> {
         tls_collect_query(
             region_id,
             peer,
-            Key::from_raw(start_key).as_encoded(),
-            Key::from_raw(end_key).as_encoded(),
+            start_key.as_encoded(),
+            end_key.as_encoded(),
             reverse_scan,
         );
         tls_collect_read_flow(
             self.req_ctx.context.get_region_id(),
-            Some(start_key),
-            Some(end_key),
+            Some(start_key.as_encoded()),
```

Comment:
Is it introduced by #12080? We also need to pick  it to other release

Reply:
Do we need to add any tests about it?

Reply:
Does it affect the hot-region schedule?

Reply:
> Does it affect the hot-region schedule?

I think it will only effect bucket bound, not hot-region-scheduler flow statistic or load-base-split key.

Reply:
> Do we need to add any tests about it?



> Does it affect the hot-region schedule?
yes，it cause that the bucket's statistics is zero. some big hot region can't splitted if it can't find the hot buckets.

Reply:
> Do we need to add any tests about it?

yes， I will add IT for it